### PR TITLE
Completed the todo app with client and server services

### PR DESCRIPTION
**Pull request by AdithyaBhat17**

### CODE REVIEW

**Comment by Shrinidhikulkarni7** [comment]
Do you require two different yml files?
Any major change between them?

**Comment by AdithyaBhat17** [comment]
@Shrinidhikulkarni7 the dev compose file runs the client on dev mode. The docker-compose.yml will create a production build (with static html, css, and js files).

Is there a way we can do this in a single file reading NODE_ENV?

File: todo-app/docker-compose-dev.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      - web
+  client:
+    build:
+      context: client/
+      dockerfile: Dockerfile
+    container_name: client
+    volumes:
+      - ./client:/app/client
+    restart: always
+    command: sh -c "yarn && yarn dev"
```
Comment (AdithyaBhat17) [inline todo-app/docker-compose-dev.yml:26]: @Shrinidhikulkarni7 in docker-compose.yml, we do `yarn && yarn build && yarn start` which builds and serves static files.

**Comment by Shrinidhikulkarni7** [comment]
> Is there a way we can do this in a single file reading NODE_ENV?

We can do it while running the compose file, but if there are multiple changes in yml file you can go with two.

**Pull request merged**